docs/backend/guides/ghc/ghc-invoicing-packages.md

```diff
@@ -216,50 +216,34 @@ mymove/pkg/services
 │   └── table_from_slice_creator_test.go
 ```
 
-## Pre-GHC MilMove FuelPrice
-
-:::caution TODO: Add description
-
-* Add **description** for this package/section.
-* After description is added, filenames (unless it is useful to leave here) can be removed.
-* Leave directory names in section.
-* Be sure to leave filename if the whole package doesn't pertain to invoicing. Leaving only the filenames that
-are important to invoicing. (E.g., `bin` and `cmd` packages. )
-
-:::
-
-
-```
-mymove/pkg/services
-├── fuelprice
-│   ├── diesel_fuel_price_storer.go
-│   ├── diesel_fuel_price_storer_test.go
-│   └── fuel_price_service_test.go
-```
-
 ## [MilMove GHC FuelPrice](https://github.com/transcom/mymove/tree/master/pkg/services/ghcdieselfuelprice)
 
-:::caution TODO: Add description
-
-* Add **description** for this package/section.
-* After description is added, filenames (unless it is useful to leave here) can be removed.
-* Leave directory names in section.
-* Be sure to leave filename if the whole package doesn't pertain to invoicing. Leaving only the filenames that
-are important to invoicing. (E.g., `bin` and `cmd` packages. )
-
-:::
-
+The GHC FuelPrice service is utilized by the milmove-task command `save_ghc_fuel_price_data.go`. It calls the EIA API v2.1.0 to retrieve the weekly No. 2 diesel fuel price for the U.S. You can access their [technical documentation here](https://www.eia.gov/opendata/documentation.php). With version 2.1.0 the EIA API added the `seriesid` endpoint. This allows us to use the `series_id` url parameter that we used in the v1 api as a keyword path parameter. The specific path we use is `https://api.eia.gov/v2/seriesid/PET.EMD_EPD2D_PTE_NUS_DPG.W`. 
```

Comment:
I think one of the most difficult things is about stuff like this is to get a general idea of how/why it's used.
Is it worth adding a sentence or two about how it's run in MilMove (called from an ECS task) and why (to inform service item pricing)? Or is that maybe outside the scope of this documentation?

Reply:
That makes sense to me. I'll add it.